modules/auxiliary/scanner/http/azure_ad_login.rb

```diff
@@ -161,6 +161,9 @@ def check_login(targeturi, domain, username, password)
     elsif auth_details.start_with?('AADSTS50053') # Account is locked
       print_error('Account is locked, consider taking time before continuuing to scan!')
```

Comment:
Does this line need the domain + username as well? 👀 

```suggestion
      print_error('#{domain}\\#{username} is locked, consider taking time before continuing to scan!')
```